pilot/pkg/serviceregistry/external/conversion.go

```diff
@@ -129,6 +129,8 @@ func convertEndpoint(service *model.Service, servicePort *networking.Port,
 			Port:        int(instancePort),
 			ServicePort: convertPort(servicePort),
 			Network:     endpoint.Network,
+			Locality:    endpoint.Locality,
+			LbWeight:    endpoint.Weight,
```

Comment:
Also update Consul and Kube. Both have the notion of Availability zone. Its currently assigned to the ServiceInstance, but that value can be set in the Locality field (after conversion to / separated format).

Reply:
Done